src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
```

Comment:
as this is an Enum, and it contains mainly constants only, and we know that constants names must be written using only uppercase letters

```suggestion
    BLACK,
    WHITE,
    RED,
    BLUE
```